pkg/packagekit/updates.scss

```
@@ -153,6 +153,13 @@ div.changelog {
   }
 }
 
+.error-log {
+  margin: 0 auto;
+  padding: 0 3ex;
+  max-width: fit-content;
+  width: 100%;
+}
```

Is there not a PF way to handle the alignment? (Perhaps you're adding this because the import doesn't seem to be done correctly somehow, so the PF CSS isn't being loaded.)

Spacing should use PF variables, so it's consistent (and would automatically update if PF ever did). I think `md` might be the right one here.

I'm not sure if max-width of fitting content and having the width at 100% makes sense here. The block is already being center aligned, so it will distribute the space on the sides (hence not needing fit-content). It might be better to let it auto-size by default and set the max-width to 100% so smaller sizes would wrap.

So, here's my suggestion to the changes:

```suggestion
.error-log {
  margin: 0 auto;
  padding: 0 var(--pf-globel--spacer--md);
  max-width: 100%;
}
```

(This might differ if the PF compoenent changes things when it's fully imported.)

---

@garrett any idea how to import in CSS

---

I meant for these commits to land as a part of my full review, not ahead of time. Whoops!

But I have a suggestion in the review. (Look below.) It changes the input and adds `<CodeBlock>` as a wrapper.

---

Solved after wrapping in CodeBlockCode in CodeBlock